ClientSyncComponent/ClientSyncComponent.Client/PuzzleEntry.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Azure;
+using Azure.Data.Tables;
+
+namespace ClientSyncComponent.Client
+{
+    /// <summary>
+    /// Azure Table entity for puzzle state
+    /// </summary>
+    public class PuzzleEntry : ITableEntity
```

Comment:
since `class="puzzle-entry"` means the entire puzzle, can this be renamed to something like `PuzzleChangeRecord`?

Reply:
Done